modules/entity/spec/sorted_state_adapter.spec.ts

```diff
@@ -316,7 +316,7 @@ describe('Sorted State Adapter', () => {
     );
 
     expect(withUpdates).toEqual({
-      ids: [AClockworkOrange.id, TheGreatGatsby.id],
+      ids: [TheGreatGatsby.id, AClockworkOrange.id],
```

Comment:
That's strange, locally the test turns green.
The order is important here (`sortComparer: (a, b) => a.title.localeCompare(b.title)`), and should be sorted as before because "a" comes before "t".

It seems like the CI didn't run these tests (this is something that I've experienced before when only specs are modified). 

```
#!/bin/bash -eo pipefail
yarn nx affected:test --base=$(cat ~/project/master.txt) --head=$CIRCLE_SHA1
yarn run v1.22.5
$ nx affected:test --base=f214eaf2 --head=c08dfc1f3aa97a0f08975bf6494b300ff5c83b88

>  NX  No projects with "test" were run


———————————————————————————————————————————————

>  NX   SUCCESS  Running target "test" succeeded



>  NX   CLOUD  See run details at https://nx.app/runs/tEAd0HkwxNR

Done in 15.30s.
CircleCI received exit code 0
```

Reply:
The test is failing locally for me.

What are you running locally?
Output of `nx report`:
```
  Node : 14.18.1
  OS   : win32 x64
  yarn : 1.22.11

  nx : 13.2.0-beta.1
  @nrwl/angular : 13.2.0-beta.1
  @nrwl/cli : 13.2.0-beta.1
  @nrwl/cypress : 13.2.0-beta.1
  @nrwl/devkit : 13.2.0-beta.1
  @nrwl/eslint-plugin-nx : 13.2.0-beta.1
  @nrwl/express : Not Found
  @nrwl/jest : 13.2.0-beta.1
  @nrwl/linter : 13.2.0-beta.1
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 13.2.0-beta.1
  @nrwl/nx-cloud : 12.5.2
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 13.2.0-beta.1
  @nrwl/web : Not Found
  @nrwl/workspace : 13.2.0-beta.1
  @nrwl/storybook : 13.2.0-beta.1
  @nrwl/gatsby : Not Found
  typescript : 4.4.4
```

Reply:
Seems like the same versions.
I'm using the following command to run the tests (on master).

```
> nx run entity:test --noCache
 PASS   Entity  modules/entity/schematics/ng-add/index.spec.ts
 PASS   Entity  modules/entity/migrations/6_0_0/index.spec.ts
 PASS   Entity  modules/entity/spec/sorted_state_adapter.spec.ts
 PASS   Entity  modules/entity/spec/unsorted_state_adapter.spec.ts
 PASS   Entity  modules/entity/spec/entity_state.spec.ts
 PASS   Entity  modules/entity/spec/state_selectors.spec.ts
 PASS   Entity  modules/entity/spec/utils.spec.ts

Test Suites: 7 passed, 7 total
Tests:       65 passed, 65 total
```

```
  Node : 16.13.0
  OS   : win32 x64
  yarn : 1.22.5

  nx : 13.2.0-beta.1
  @nrwl/angular : 13.2.0-beta.1
  @nrwl/cli : 13.2.0-beta.1
  @nrwl/cypress : 13.2.0-beta.1
  @nrwl/devkit : 13.2.0-beta.1
  @nrwl/eslint-plugin-nx : 13.2.0-beta.1
  @nrwl/express : Not Found
  @nrwl/jest : 13.2.0-beta.1
  @nrwl/linter : 13.2.0-beta.1
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 13.2.0-beta.1
  @nrwl/nx-cloud : 12.5.2
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 13.2.0-beta.1
  @nrwl/web : Not Found
  @nrwl/workspace : 13.2.0-beta.1
  @nrwl/storybook : 13.2.0-beta.1
  @nrwl/gatsby : Not Found
  typescript : 4.4.4
```

Reply:
You're on Node.js 16.13 though 🤔